File: binary_search/problems/old/src/findMin.go
package main

import (
	"fmt"
)

func main() {
	nums := []int{2, 2, 2, 0, 1}
	min := findMin(nums)
	fmt.Println("Minimum element in: ", nums, " is: ", min)
}

func findMin(arr []int) int {
	min := arr[0]
	for _, num := range arr {
		if num < min {
			min = num
		}
	}
	return min
}

File: binary_search/problems/old/findMin.js
function findMin(nums) {
  let minVal = nums[0];
  for (let num of nums) {
    if (num < minVal) {
      minVal = num;
    }
  }
  return minVal;
}

const arr = [2, 2, 2, 0, 1];

console.log(findMin(arr))